.github/workflows/integration-tests.yml

```diff
@@ -0,0 +1,40 @@
+name: Bulk Integration Test
+
+on: [
+  workflow_dispatch,
+  # Uncomment below to automatically run on pushes to branches
+  # push,
```

Comment:
Can this comment be removed? I don't think we will want to run tests for 10 time automatically on each push.

Reply:
Added this as a hint in cases where folks want to quickly iterate automatically rather than using the GitHub UX if they are troubleshooting an issue. I would like to leave the comment in, what do you think?

Reply:
I think a quick click on the button to trigger the workflow will be easy enough. One can easily trigger the workflow AS NEEDED by clicking the button. 
If one has to run this workflow on every push, that would mean that the test reliability is too low to continue development. We wouldn't want to see that happens.